app/scripts/views/components/dataTable/tableViewTemplate.js

```diff
@@ -12,6 +12,7 @@
     ':display-name="displayName" :chart-ctrl="chartInst"' +
     ':has-chart-title="true" :groupid="attributes.group_id" ' +
     ':reset-btn-id="resetBtnId" :chart-id="chartId" :attributes="attributes" ' +
+    ':show-survival-icon.sync="showSurvivalIcon"' +
```

Comment:
show we have button description of this some thing like `alt ="rainbow survival"`